packages/core/src/app/order/OrderSummary.tsx

```diff
@@ -48,6 +53,25 @@ const OrderSummary: FunctionComponent<OrderSummaryProps & OrderSummarySubtotalsP
                     storeCurrencyCode={storeCurrency.code}
                 />
             </OrderSummarySection>
+
+            {isTaxIncluded && <OrderSummarySection>
+                <h5
+                    className="cart-taxItem cart-taxItem--subtotal optimizedCheckout-contentPrimary"
+                    data-test="tax-text"
+                >
+                    <TranslatedString
+                        id="tax.inclusive_label"
+                    />
+                </h5>
+                {(taxes || []).map((tax, index) => (
```

Comment:
If `taxes` is empty, do we still need to display the whole  `<OrderSummarySection>`?

Reply:
Good point, just didn't imagine a world without tax 😅